staging/src/k8s.io/apimachinery/pkg/util/sets/string.go

```diff
@@ -28,7 +28,7 @@ type String map[string]Empty
 
 // NewString creates a String from a list of values.
 func NewString(items ...string) String {
-	ss := String{}
+	ss := make(String, len(items))
```

Comment:
Could do the same optimization on line 40

Reply:
Sure. Updated.